Lombiq.Analyzers.NetFx/Lombiq.Analyzers.NetFx.nuspec.template

```diff
@@ -5,8 +5,7 @@
         <version>$version$</version>
         <title>Lombiq .NET Analyzers for .NET Framework</title>
         <authors>Lombiq Technologies</authors>
-        <license type="file">License.md</license>
-        <licenseUrl>https://aka.ms/deprecateLicenseUrl</licenseUrl>
+        <license type="expression">BSD-3-Clause</license>
```

Comment:
Are still both needed, the license in the template and the csproj?

Reply:
Ah, no it seems this is no longer needed.
Test run: https://github.com/Lombiq/.NET-Analyzers/actions/runs/9470163660
Test release of NetFx package without the license expression in the csproj: https://www.nuget.org/packages/Lombiq.Analyzers.NetFx/5.0.1-alpha.3.osoe-852

Have updated it for all of the packages now